res/wp-config.tpl.php

```diff
@@ -22,26 +22,18 @@
 /**
  * Require Composer's autoloader.
  */
-require_once __DIR__ . '/{{{VENDOR_DIR}}}/autoload.php';
+require_once __DIR__ . '/___WP_CONFIG_VENDOR_DIR___/autoload.php';
```

Comment:
Switched to a constant-like placeholder to prevent a syntax error in this file when the same style was used for `___WP_CONFIG_ENV_PATHS___ `.